### ISSUE TITLE
extbans

### ISSUE THREAD

**Issue by DanielOaks** [DanielOaks/oragono#69]
There are some nice ones out there that would be useful. Finish the ircdocs list first and refer to that.

**Comment by slingamn** [comment]
Since accounts are a first-class concept in oragono, should we get different/alternative syntax for account whitelists / blacklists? (Maybe keep the freenode-style extban syntax for `/mode` / `MODE`, but provide an easier-to-use interface via chanserv?)

**Comment by DanielOaks** [comment]
Yeah that makes sense. Laid out my thoughts over here: https://github.com/oragono/oragono/issues/221#issuecomment-462491986

**Comment by slingamn** [comment]
The quiet channel mode (#307) will become an extban

**Comment by slingamn** [comment]
jesopo mentions this in connection with quiet: https://gist.github.com/jesopo/2a3c119ac85f722d61bd4783271338e0

**Comment by slingamn** [comment]
I thought about this and I'm not sure I want to prioritize it anymore. In particular, the objective of #749 is to converge the nickname and account concepts; as part of it, it would make sense to add a mode where if you're logged in, your nickname must equal your account name. This would make ordinary nickname bans work on accounts.

**Comment by slingamn** [comment]
Tentatively removing this from the 2.1 milestone in favor of #749.
